app/build.gradle

```diff
@@ -82,13 +82,6 @@ play {
     track = "${track}"
 }
 
-repositories {
-    mavenCentral()
-    maven { url 'https://github.com/lijingle1/stetho-realm/raw/master/maven-repo' }
-    maven { url 'http://dl.bintray.com/amulyakhare/maven' }
-    maven { url "https://clojars.org/repo/" } //for icepick.
```

Comment:
Moving these definition is not related to introducing widget-fontawesome. 
However, I think moving these in root build.gradle is simpler and good.